src/lib/core/components/Form/hooks/useMutators.ts

```diff
@@ -67,9 +67,7 @@ export const useMutators = (externalMutators?: DynamicFormMutators) => {
                 ...(mutators.errors
                     ? {errors: mergeValuesOrErrors(store.errors, mutators.errors)}
                     : {}),
-                ...(mutators.values
-                    ? {values: mergeValuesOrErrors(store.values, mutators.values)}
-                    : {}),
```

Comment:
Change to the code `...(mutators.values ? {values: mutators.values} : {})` cannot be used because the data format will be incorrect. Therefore, I am throwing an empty object into `mergeValuesOrErrors` as the source object.